blueprint/abac/client-devices.md

```diff
@@ -0,0 +1,8011 @@
+1.  Within an administrative PowerShell session navigate to the install directory of the application
+2.  Run the following command
+```powershell
+$whitelistlevel = {the chosen method of whitelist e.g. hash}
+$Outputlocation = {the path to output the policy file}
+$fallbacklevel = {the backup method of whitelist e.g. publisher}
+$directory = {the path to be scanned}
+New-CIPolicy -Level $whitelistlevel -FilePath $Outputlocation -Fallback $fallbacklevel  -ScanPath $directory -UserPEs
+```
+3. Convert the new base policy to a supplementary policy.
+
+#### Whitelisting all errors in the event log
+
+Applications which are not whitelisted will be logged in the code integrity event log on execution. These applications can be whitelisted using the following procedure:
+
+1.  Within an administrative PowerShell session run the following command
+```powershell
+$whitelistlevel = {the chosen method of whitelist e.g. hash}
+$Outputlocation = {the path to output the policy file}
+$fallbacklevel = {the backup method of whitelist e.g. publisher}
+New-CIPolicy -Level $whitelistlevel -FilePath $Outputlocation -Fallback $fallbacklevel  -audit -UserPEs
+```
+2. Convert the new base policy to a supplementary policy.
+
+
+#### Linking to the base policy
+
+To convert a base policy to a supplementary policy of another base policy they must be linked. Upon linking the policyID of the supplementary policy will be set to a new GUID. This new guid is required when deploying the supplementary policy via Microsoft Endpoint Manager. 
+
+The procedure to link the policies is as follows:
+1. Using an administrative PowerShell session run
+```powershell
+$SupWDACPolicy = {path to the supplementary policy file}
+$SupWDACPolicyName = {Name of the supplementary policy}
+$WDACPolicy = {path to the policy file}
+$WDACPolicyID = {base policy id available within the base policy PolicyID tags}
+Set-CIPolicyIdInfo -FilePath $SupWDACPolicy -PolicyName $SupWDACPolicyName -SupplementsBasePolicyID $WDACPolicyID -BasePolicyToSupplementPath $WDACPolicy
+```
+
+### WDAC Policy - Policy Signing
+
+Prior to deployment of the WDAC policy it can be signed using an internal Certificate Authority code signing certificate or a purchased code signing certificate.
+
+The procedure to sign the policy is as follows:
+* Using an administrative PowerShell session run
+```powershell
+Add-SignerRule -FilePath {Path to the XML policy file} -CertificatePath {Path to exported .cer certificate} -Kernel -User –Update
+ConvertFrom-CIPolicy -XmlFilePath {Path to the XML policy file} -BinaryFilePath `Binary output location`
+{Path to signtool.exe} sign -v /n {Certificate Subject name} -p7 . -p7co 1.3.6.1.4.1.311.79.1 -fd sha256 {Binary policy location}
+```
+
+### WDAC Policy - Deployment
+
+The above policies are deployed through Microsoft Endpoint Manager/Intune. The deployment configuration is available within the [Intune Configuration](/blueprint/abac/intune-configuration.html#agency-wdacenablement) section.
```

Comment:
Link destination doesn't exist, could be missing?

Reply:
updated. It was pointing to a section i removed.